week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -53,9 +53,27 @@ console.log(arrayLengths);
 /* + + + + + + + */
 /* Now try the exercise */
 
-function getEligibleStudents() {
+function getEligibleStudents(twoDArr) {
+
+  function attendedMinEightClasses(listOfNames){
+    return listOfNames[1] >= 8;
+  }
+  
+  let result = twoDArr.filter(attendedMinEightClasses);
+
+  let requiredStudentNames = result.map(function (sortNames){
+
+    for(let i = 0; i < sortNames.length; i++){
```

Comment:
Here you're trying to get the first item out of `sortNames` with a `for` loop. However, you don't need to use a `for` loop for this one - there is a slightly simpler way of doing it